adapters/repos/db/vector/ssdhelpers/product_quantization.go

```diff
@@ -358,13 +359,20 @@ func (pq *ProductQuantizer) Fit(data [][]float32) {
 }
 
 func (pq *ProductQuantizer) Encode(vec []float32) []byte {
-	codes := make([]byte, pq.m)
+	codes := make([]byte, pq.m+4)
```

Comment:
Should these Distortion changes be in this commit?

Reply:
Definitely not... Removed already.